src/track/beats.h

```diff
@@ -38,18 +38,11 @@ namespace mixxx {
 class Beats final : public QObject {
     Q_OBJECT
   public:
-    /// iSampleRate may be provided if a more accurate sample rate is known
-    /// than the one associated with the Track.
+    /// Initialize beats with only the track pointer.
     explicit Beats(const Track* track);
```

Comment:
What beats does the class contain if this constructor is used? Are there any beats?

Reply:
Nope. They can only be set with the `setGrid` method afterwards.